options/db_options.cc

```diff
@@ -1056,6 +1070,34 @@ void MutableDBOptions::Dump(Logger* log) const {
                    compaction_readahead_size);
   ROCKS_LOG_HEADER(log, "                 Options.max_background_flushes: %d",
                           max_background_flushes);
+  ROCKS_LOG_HEADER(log, "Options.daily_offpeak_start_time_utc: %s",
+                   daily_offpeak_start_time_utc.c_str());
+  ROCKS_LOG_HEADER(log, "Options.daily_offpeak_end_time_utc: %s",
+                   daily_offpeak_end_time_utc.c_str());
+}
+
+bool MutableDBOptions::IsNowOffPeak(SystemClock* clock) const {
```

Comment:
Not sure if this is the best place for this util function. I can create a new util, maybe call it `time_util` or something else. Please let me know.

Also, I am still a C++ n00b. If there's a better way to do what I did here, please let me know.